install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -19,6 +19,20 @@ data:
     disablePolicyChecks: true
     {{- end }}
 
+    {{- if .Values.mixer.telemetry.reportBatchMaxEntries }}
+    # Set reportBatchMaxEntries to 0 to use the default batching behavior. A positive
+    # value indicates the number of requests that are batched before telemetry data 
+    # is sent to the mixer server
+    reportBatchMaxEntries: {{ .Values.mixer.telemetry.reportBatchMaxEntries }}
+    {{- end }}
+
+    {{- if .Values.mixer.telemetry.reportBatchMaxTime }}
+    # Set reportBatchMaxTime to 0 to use the default batching behavior. A positive time 
```

Comment:
what is the default time?   it would be good to share in the doc.

Reply:
The default behavior changes from version to version. In 1.0.x, it was every 1000 requests or 1 second. In 1.1.x, it is every 100 requests or 1 second. I'm a bit reluctant to put a value here as I've seen a few places that code and comment get out of sync, which could be confusing. I could put a link to the code where the default behavior is specified, but if the code moves, the comment again will be confusing. Let me know if you have some suggestions here without creating unnecessary dependencies.

Reply:
This comment in only for a given release since it is for master, it is the value for master :)  I just think we will get asked for the default...   it is probably a good to put a comment in code that this value is configurable now with default documented.

Reply:
oh you mean comments for the doc? I might have misunderstood. afaik, docs are generated from `istio/api`. See comments here: https://github.com/istio/api/blob/master/mixer/v1/config/client/client_config.proto#L149-L159. Is that sufficient or would you rather we explicitly put "every 100 request or 1 second" as a the comment in configmap.yaml?

Reply:
ok, explicitly added the default behavior in the comment section of mixer/values.yaml